components/form/builder/src/Standard/Fields/Multifield/index.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types'
+
+const MultiField = ({
+  field,
+  tabIndex,
+  onChange,
+  onFocus,
+  onBlur,
+  errors,
+  alerts,
+  renderer
+}) => {
+  const {fields} = field || {}
+
+  const rendererResponse = renderer({
+    id: field.id,
+    type: field.type,
+    innerProps: fields.map(props => ({
```

Comment:
IMHO should be called with ProxyField so that each field can determine its own representation

Reply:
I like it! Thanks @javiauso